.github/workflows/ci.yml

```diff
@@ -74,15 +74,19 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # checkout tags so version in Manifest is set properly
       - uses: actions/setup-java@v1
         with:
           java-version: 11
       - name: Maven Install
         run: |
           export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
-          ./bin/retry ./mvnw install ${MAVEN_FAST_INSTALL} -pl '!presto-test-jdbc-compatibility-old-driver,!presto-docs,!presto-server,!presto-server-rpm'
-      - name: Test JDBC clients
+          ./bin/retry ./mvnw install ${MAVEN_FAST_INSTALL} -pl '!presto-test-jdbc-compatibility-old-driver,!presto-test-jdbc-compatibility-old-server,!presto-docs,!presto-server,!presto-server-rpm'
+      - name: Test old JDBC vs current server
         run: presto-test-jdbc-compatibility-old-driver/bin/run_tests.sh
+      - name: Test current JDBC vs old server
+        run: ./mvnw test -B -Dair.check.skip-all -pl presto-test-jdbc-compatibility-old-server
```

Comment:
Since this is a "normally testable module", it doesn't need to be included here (a special case).

You can add this to `test` job (regular module tests).

Reply:
It feels natural to me to keep those two together. Even if technically it is possible to run it somewhere else. I would keep it.

Reply:
OK